Formula/shibboleth-sp.rb

```diff
@@ -43,7 +48,19 @@ def install
     system "make", "install"
   end
 
-  plist_options :startup => true, :manual => "shibd"
+  def caveats
+    mod = (build.with? "apache-22")? "mod_shib_22.so" : "mod_shib_24.so"
+    <<-EOS.undent
+      You must manually edit httpd.conf to include
+      LoadModule mod_shib #{lib}/shibboleth/#{mod}
+      You must also manually configure
+        #{etc}/shibboleth/shibboleth2.xml
+      as per your own requirements. For more information please see
+        https://wiki.shibboleth.net/confluence/display/EDS10/3.1+Configuring+the+Service+Provider
```

Comment:
Let's leave these as they contain Homebrew-specific paths but the instructions aren't Homebrew-only.